extensions/ql-vscode/src/databases/db-manager.ts

```diff
@@ -1,22 +1,19 @@
-import { logger } from '../logging';
 import { DbConfigStore } from './db-config-store';
 import { DbItem } from './db-item';
+import { createLocalTree, createRemoteTree } from './db-tree-creator';
 
 export class DbManager {
   constructor(
     private readonly dbConfigStore: DbConfigStore
   ) {
   }
 
-  public loadDatabases(): void {
+  public getDbItems(): DbItem[] {
```

Comment:
Minor: When I update my config file, the treeview doesn't update until I've restarted VS Code.

(Feel free to ignore me if this is expected and we plan to hook up the watcher at a later stage!)

Reply:
Yep, we'll hook to the watcher in the future :)